static/styles/settings.css

```diff
@@ -626,6 +626,7 @@ input[type="checkbox"] {
     margin: 10px 0;
 }
 
+.add-new-emoji-box,
```

Comment:
This is a misresolved merge conflict; b7335d0dec00de426ee5f5aeba3653efca35b729 removed that line.

Please be careful to self-review for this type of mistake when resolving merge conflicts; it's an easy one to make and is a very common cause of merging changes breaking another part of the product.